src/test/utils/findOrCreatePlugin.js

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Mongoose findOrCreate Plugin
+ * Copyright(c) 2012 Nicholas Penree <nick@penree.com>
+ * MIT Licensed
+ */
+
+// Inlined mongoose-findorcreat until this fixed is merged: https://github.com/drudge/mongoose-findorcreate/pull/18
```

Comment:
Inlined mongoose-findorcreate until this fixed is merged: https://github.com/drudge/mongoose-findorcreate/pull/18